### PR TITLE
Deprecate zh_TW methods that only call base methods

### PR DESCRIPTION
**Pull request by krsriq**
### What is the reason for this PR?

In `zh_TW` there are two classes that use `Factory::create('en_US')` only to call base methods.

- [ ] A new feature
- [ ] Fixed an issue (resolve #ID)

### Author's checklist

- [x] Follow the [Contribution Guide](https://github.com/FakerPHP/Faker/blob/main/.github/CONTRIBUTING.md)
- [ ] New features and changes are [documented](https://github.com/FakerPHP/fakerphp.github.io)

### Summary of changes

This PR deprecates these classes and methods. For BC the respective parent methods are called.

### Review checklist

- [ ] All checks have passed
- [ ] Changes are approved by maintainer

### CODE REVIEW

**Comment by bramceulemans** [comment]
Looks good to me.

**Review by bramceulemans — APPROVED** [review]

File: src/Faker/Provider/zh_TW/Payment.php [hunk 1]
```diff
@@ -2,10 +2,20 @@
...
 
+/**
+ * @deprecated Use {@link \Faker\Provider\Payment} instead
+ * @see \Faker\Provider\Payment
+ */
 class Payment extends \Faker\Provider\Payment
 {
+    /**
+     * @return array
+     * @throws \Exception
```
Comment (GrahamCampbell) [inline src/Faker/Provider/zh_TW/Payment.php:13]: Why? Is there similar doc on the other creditCardDetails methods? If not, we should not add this.

**Review by GrahamCampbell — CHANGES_REQUESTED** [review]

File: src/Faker/Provider/zh_TW/Payment.php (hunk 1 above)
Comment (krsriq) [reply src/Faker/Provider/zh_TW/Payment.php]: There's only the core method and that has a `@return` docblock, but no `@throws` - which I now removed here. Also removed the new docblock tags in `zh_TW\Internet`.

Comment (pimjansen) [reply src/Faker/Provider/zh_TW/Payment.php]: @GrahamCampbell can you do final review so we can close?

**Review by pimjansen — APPROVED** [review]

**Comment by krsriq** [comment]
Thanks!

**Pull request merged**